persimmon/persimmon/settings.py

```diff
@@ -23,7 +23,7 @@
 SECRET_KEY = ')dimn^b^qcbpf8stkoinv#5b$&_@@y4!+2v_zys5pt&8ky1ade'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```

Comment:
We should leave this `True` until it's time to deploy to production

Reply:
will do